src/js/components/Box.js

```diff
@@ -26,6 +27,7 @@ var Box = React.createClass({
     ]),
     reverse: React.PropTypes.bool,
     responsive: React.PropTypes.bool,
+    primary: React.PropTypes.bool,
```

Comment:
Why are we adding this to Box instead of Article or Section? I have thought of Box as a generic container used in many places, such as Menu, Header, Footer, etc.